components/count-badge/demo/count-badge.html

```diff
@@ -61,12 +67,7 @@ <h2>Small Notification Badge with Tooltip</h2>
 					<d2l-count-badge size="small" has-tooltip text=" 2 new notifications." type="notification" number="2"></d2l-count-badge>
 				</template>
 			</d2l-demo-snippet>
+
 		</d2l-demo-page>
-		<script type="module">
```

Comment:
This was moved because it was causing a JavaScript error in Safari due to the timing of this script executing and the demo template being stamped. An alternative would also be to wrap this in a `setTimeout` to defer its execution.